### PR TITLE
CHANGELOG: Document changes since v0.9.0

### PR DESCRIPTION
**Pull request by wking**
Through 63bdb7fc (#1050).

CC @abhinavdahiya, @crawford.

### CODE REVIEW

**Comment by abhinavdahiya** [comment]
/approve

looks good.

**Comment by wking** [comment]
[e2e-aws][1]:

```
6 error level events were detected during this test run:

Jan 15 01:30:16.732 E kube-apiserver Kube API started failing: Get https://ci-op-5979kimc-1d3f3-api.origin-ci-int-aws.dev.rhcloud.com:6443/api/v1/namespaces/kube-system?timeout=3s: context deadline exceeded (Client.Timeout exceeded while awaiting headers)
Jan 15 01:34:20.521 E kube-apiserver Kube API started failing: Get https://ci-op-5979kimc-1d3f3-api.origin-ci-int-aws.dev.rhcloud.com:6443/api/v1/namespaces/kube-system?timeout=3s: context deadline exceeded
Jan 15 01:34:47.732 E kube-apiserver Kube API started failing: Get https://ci-op-5979kimc-1d3f3-api.origin-ci-int-aws.dev.rhcloud.com:6443/api/v1/namespaces/kube-system?timeout=3s: context deadline exceeded (Client.Timeout exceeded while awaiting headers)
Jan 15 01:34:48.732 - 30s   E kube-apiserver Kube API is not responding to GET requests
Jan 15 01:35:17.733 E kube-apiserver Kube API started failing: Get https://ci-op-5979kimc-1d3f3-api.origin-ci-int-aws.dev.rhcloud.com:6443/api/v1/namespaces/kube-system?timeout=3s: context deadline exceeded (Client.Timeout exceeded while awaiting headers)
Jan 15 01:39:19.732 E kube-apiserver Kube API started failing: Get https://ci-op-5979kimc-1d3f3-api.origin-ci-int-aws.dev.rhcloud.com:6443/api/v1/namespaces/kube-system?timeout=3s: context deadline exceeded (Client.Timeout exceeded while awaiting headers)
```

/retest

[1]: https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_installer/1064/pull-ci-openshift-installer-master-e2e-aws/2911

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,114 @@ All notable changes to this project will be documented in this file.
...
+  interfaces if Neutron supports them to support future Kuryr
+  integration.
+- On OpenStack, masters and workers have been consolidated in a single
+  subnet to simplify the deployment.
+- On OpenStack, the Ignition security group now only allows internal
+  connections, and no longer allows connections from outside the
+  cluster network.
+- On OpenStack, the machine(set) templates have been updated to set
+  `cloudName` and some other properties.
+- On libvirt, `destroy cluster` is not more robust in the face of
```
Comment (crawford) [inline CHANGELOG.md]: s/not/now/

Comment (wking) [reply CHANGELOG.md]: > s/not/now/

Thanks :).  Fixed with 2cbe0f8 -> 997f778.  Can you drop a fresh `/lgtm`?

**Comment by wking** [comment]
I've pushed 997f778 -> 68692df, bumping the release date to today.  @crawford, can I get a fresh `/lgtm`?

**Comment by abhinavdahiya** [comment]
/lgtm

> I've pushed [997f778](https://github.com/openshift/installer/commit/997f77885144eebdab14fa777bc98755fbbe26c1) -> [68692df](https://github.com/openshift/installer/commit/68692df72f623e79fec933ad6e849bed622901b3), bumping the release date to today. @crawford, can I get a fresh `/lgtm`?

**Comment by wking** [comment]
/hold cancel

We're about to tag 0.10.0 on top of this via #1068.

**Comment by wking** [comment]
[e2e-aws][1]:

```
fail [k8s.io/kubernetes/test/e2e/storage/pvc_protection.go:61]: Failed waiting for PVC to be bound PersistentVolumeClaim pvc-mw6ll not in phase Bound within 3m0s
Expected error:
    <*errors.errorString | 0xc4210aae80>: {
        s: "PersistentVolumeClaim pvc-mw6ll not in phase Bound within 3m0s",
    }
    PersistentVolumeClaim pvc-mw6ll not in phase Bound within 3m0s
not to have occurred
```

and two other volume-related errors.

/retest

[1]: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_installer/1064/pull-ci-openshift-installer-master-e2e-aws/2918/build-log.txt

**Comment by wking** [comment]
[e2e-aws error][1] was:

```
level=error msg="Error: Error applying plan:"
level=error
level=error msg="1 error occurred:"
level=error msg="\t* module.vpc.aws_lb.api_external: 1 error occurred:"
level=error msg="\t* aws_lb.api_external: timeout while waiting for state to become 'active' (timeout: 10m0s)"
```

[1]: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_installer/1064/pull-ci-openshift-installer-master-e2e-aws/2926/build-log.txt

**Comment by wking** [comment]
[e2e-aws][1]:

```
level=error msg="Error: Error applying plan:"
level=error
level=error msg="1 error occurred:"
level=error msg="\t* module.vpc.aws_lb.api_internal: 1 error occurred:"
level=error msg="\t* aws_lb.api_internal: timeout while waiting for state to become 'active' (last state: 'provisioning', timeout: 10m0s)"
```

Sigh.

/retest

[1]: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_installer/1064/pull-ci-openshift-installer-master-e2e-aws/2937/build-log.txt

**Comment by wking** [comment]
woo, finally

**Pull request merged**